dd_game.yml

```diff
@@ -0,0 +1,184 @@
+name: dd_game
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - _libgcc_mutex=0.1=main
+  - _openmp_mutex=4.5=1_gnu
+  - attrs=21.2.0=pyhd3eb1b0_0
+  - autopep8=1.5.7=pyhd3eb1b0_0
```

Comment:
I'm not sure what this file is doing exactly.  I'm guessing it is programme=version= ??
What is after the final equals sign?

Reply:
i think it links to a specific branch or commit that the install of that version should link to